root/collection/CollectionEdit.js

```diff
@@ -24,7 +24,7 @@ const CollectionEdit = ({collection, collectionTypes, form}: Props) => (
     entity={collection}
     fullWidth
     page="edit"
-    title={l('Edit Collection')}
+    title={l('Edit collection')}
```

Comment:
I would be fine with just `Edit`, for the same reasons as above (about `Remove`).